js/customscripts.js

```diff
@@ -29,8 +29,35 @@ $( document ).ready(function() {
             easing: 'linear'
         }
     });
+
+    initFeedbackForm();
 });
 
+function initFeedbackForm() {
+  $('.form-collapse').each(function(){
+      let container = $(this),
+          opener = container.find('.form-collapse__opener'),
```

Comment:
Suggest to add a separate class with `js-` prefix to add event listeners:

```suggestion
          opener = container.find('.js-form-collapse__opener'),
```